### PR TITLE
Create Config Uploader

### PR DESCRIPTION
**Pull request by waltaskew**
Introduce a script for uploading rainfall configuration files.

### CODE REVIEW

**Comment by waltaskew** [comment]
cc @rrsutormin @ahmustafa

**Comment by alevenberg** [comment]
A few random thoughts:
- So far, I've been using kebab case, not snake case for directories. We might want to standardize.
- I also use conventional commits (https://www.conventionalcommits.org/en/v1.0.0/)
- To avoid branch pollution on the upstream repo, I recommend making a fork.
- I don't know if main is protected on the upstream repo. I don't think I have the access to change that setting. We might need to login via USL to get that access.

File: config_uploader/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+the duration of its associated rainfall configuration file.  Note that
+the uploaded rainfall configuration files will have different names in
+GCS. They will have names in GCS with the form 'Rainfall_Data_X.txt'
+where X is some number.
+
+The resulting batch configuration file will map each rainfall
+configuration file to the CityCAT configuration file with the correct
+simulation run time.
+
+## Installation
```
[the remark this answers is not in the document]
Comment (waltaskew) [reply config_uploader/README.md:21]: this should work on windows as well

**Comment by waltaskew** [comment]
> A few random thoughts:
> 
> * So far, I've been using kebab case, not snake case for directories. We might want to standardize.

We've been using snake case for directories in all the other climateiq repos I've seen.

> * I also use conventional commits (https://www.conventionalcommits.org/en/v1.0.0/)
> * To avoid branch pollution on the upstream repo, I recommend making a fork.
> * I don't know if main is protected on the upstream repo. I don't think I have the access to change that setting. We might need to login via USL to get that access.

We should sticking to the same set of conventions across all of our repos. We don't want different rules in different repos. The SDLC I wrote out here:
https://www.notion.so/Software-Development-Life-Cycle-05b5181505cd4cd298299b6cd6c745ad

- Does not mention conventional commits and I don't think we're using them elsewhere. If this is something you'd like to make part of the SDLC, you should advocate for it publicly and get it into the document.
- Explicitly states that we're branching rather than forking. Again, if you'd like to advocate for something different, that's something to discuss with the broader team and get into the SDLC document.
- I have admin permissions on the repo so can set up branch protection and grant other people admin status.

**Pull request merged**